KMaina/Fast-Food-App
# #160648396 Order History Redesign

#### What does this PR do?
Allows a user to view their order history
#### Description of Task to be completed?
A user should be able to view the meals ordered and their status either accepted or declined.
#### How should this be manually tested?

- Clone the repo onto your local machine.

- Open a terminal and run cd Fast-Food-App

- run `git checkout -b ch-order-history-160648396`

- run `git pull https://github.com/KMaina/Fast-Food-App.git ch-order-history-160648396`

- In the folder where you cloned the repo, you should open the UI folder.

- Open the `user-order-history.html` to see it in your browser for review.

#### Any background context you want to provide?
This is a revision of the user-order-history.html page. It was done to make improvements to the UI.
#### What are the relevant pivotal tracker stories?
#160648396
